lib/service/connect.go

```diff
@@ -25,6 +25,7 @@ import (
 	"golang.org/x/crypto/ssh"
 
 	"github.com/gravitational/teleport"
+	apiclient "github.com/gravitational/teleport/api/client"
```

Comment:
```go
"github.com/gravitational/teleport/api/client"
```

Reply:
"github.com/gravitational/teleport/lib/client" is also imported